### ISSUE TITLE
[Feature request] devenv launch modmesh

### ISSUE THREAD

**Issue by tai271828** [solvcon/devenv#112]
- Ubuntu Jammy has shipped with Qt6 needed for building modmesh
- Ubuntu Focal could build modmesh smoothly by installing third-party Qt libraries from Qt's website

I think it is time to make the feature of `devenv launch modmesh`.

**Comment by yungyuc** [comment]
Does this request depend on a build script of qt6 in devenv?

**Comment by tai271828** [comment]
> Does this request depend on a build script of qt6 in devenv?
No, not yet. Building qt6 is another issue. I think I will implement this feature specific to Ubuntu Jammy, which shipped with qt6 binary by default.
